stripe/src/main/java/com/stripe/android/view/PaymentFlowConfig.java

```diff
@@ -48,10 +104,10 @@ public boolean equals(Object o) {
 
         PaymentFlowConfig that = (PaymentFlowConfig) o;
 
-        if (isHideAddressScreen() != that.isHideAddressScreen()) return false;
-        if (isHideShippingScreen() != that.isHideShippingScreen()) return false;
-        if (!getHiddenAddressFields().equals(that.getHiddenAddressFields())) return false;
-        if (!getOptionalAddressFields().equals(that.getOptionalAddressFields())) return false;
+        if (isHideShippingInfoScreen() != that.isHideShippingInfoScreen()) return false;
```

Comment:
Java bug: if you override equals, you have to override hashcode or the world will end (or at least hash maps will):
https://stackoverflow.com/questions/2265503/why-do-i-need-to-override-the-equals-and-hashcode-methods-in-java